helm/pv-cleaner-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.clusterRoleName }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - persistentvolumes
+      - persistentvolumeclaims
+      - pods
```

Comment:
Why does this need all verbs for PODs?

Reply:
Hm, true, for pods it needs only `list` and `delete`.